webpack.config.js

```diff
@@ -20,14 +22,14 @@ module.exports = {
     compress: true,
     historyApiFallback: true,
     // hot: true,
-    port: 3000,
+    port: Number(PROXY_PORT),
```

Comment:
Based on our conversation yesterday, perhaps we should rename this to `DEV_SERVER_PORT`.

> "I still don't love "PROXY_PORT". It should really be something like DEV_SERVER_PORT because it's only used in dev. The other port is used internally during dev and externally during production (in production the node server serves the static assets, in development the frontend server proxies requests to the backend. It's v. weird)" - @lukebatchelor